application/models/models.py

```diff
@@ -65,10 +70,6 @@ def rurefs(self):
     def exids(self):
         return [exercise.exercise_id for exercise in self.exercises]
 
-    @property
-    def classifiers(self):
-        return [{classifier.kind: classifier.value} for classifier in self.classifications]
-
 
 class BusinessModel(Base):
```

Comment:
It looks like `BusinessModel` has a wrong`__init__` function too. It shouldn't have `items` or `status` fields